src/Bookings.js

```diff
@@ -1,8 +1,9 @@
 import React from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
 
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
+export const results={FakeBookings}
```

Comment:
- this should be export default Bookings as you are importing Bookings on Line 7 in src/App.js,
` import Bookings from "./Bookings";`
- usually we tend to have default exports at the bottom of the file
```
const Bookings = () => {
  const search = searchVal => {
    console.info("TO DO!", searchVal);
  };

  return (
    <div className="App-content">
      <div className="container">
        <Search search={search} />
        <SearchResults results={FakeBookings} /> 
      </div>
    </div>
  );
  
  export default Bookings
```